docs/WritingTests.asciidoc

```diff
@@ -473,6 +473,33 @@ sub run () {
 }
 --------------------------------------------------------------
 
+=== Logging package versions used for test
+
+There are two set of packages that can be included in test logs:
+1. Packages installed on the worker itself - stored as `worker_packages.txt`.
+2. Packages installed on SUT - stored as `sut_packages.txt`.
+
+For both sets, if present, openQA will include difference since last good job
+in the "investigation" tab of a failed job.
+
+To enable logging worker package versions, set `PACKAGES_CMD` in `workers.ini`.  The command should print installed packages with their version to stdout. For RPM-based systems it can be for example `rpm -qa`.
+
+To enable logging SUT package versions, make the test create `sut_packages.txt` file in the current worker directory. If `upload_logs()` is used, the resulting file needs to be copied/moved.
+
```

Comment:
Please wrap these lines at a more reasonable length. I guess we are usually wrapping at 80 characters in these files.